Add ability to replay match from specified event

trarbr opened the pull request:
Closes #35

Commit:
Add ability to replay match from specified event

nicoeg reviewed (commented):
This is so cool! 🤩 

Seems like the total event count is going up as the events are being played. Would it be possible for it to load them all at the beginning?
![image](https://user-images.githubusercontent.com/947230/96919444-0e31b080-14ac-11eb-923c-185137d5f990.png)

Also could you add a comment to the readm explaining how to start it and link to one of the log files as examples?

nicoeg commented:
It could also be nice to be able to pause it but let's save that for later

nicoeg commented on assets/css/app.scss:
```diff
@@ -371,7 +371,7 @@ body {
+.match-debug {
     background: rgba(0, 0, 0, 0.3);
     color: #fff;
     padding: 20px;
```
```suggestion
    padding: 20px;
    margin-top: 20px;
```

Commit:
Fix background white fade

nicoeg reviewed (changes requested):
I've pushed a commit that fixes the white background fade issue. Hope it's okay 😛

Commit:
Add load function to immediately load all events from logfile

Commit:
Add description of load and playback functions

trarbr committed:
Update assets/css/app.scss

Co-authored-by: Nicolaj Egelund <nicolaj@uptime.dk>

trarbr commented:
> This is so cool! 🤩
> 
> Seems like the total event count is going up as the events are being played. Would it be possible for it to load them all at the beginning?
> 
> Also could you add a comment to the readm explaining how to start it and link to one of the log files as examples?

Great idea :+1: I have added `CsgoStats.load/1` to load all events at once, and then it lets you jump between events as you like to. I have also added a description of load and playback functions to the readme.

nicoeg reviewed (approved):
Cool stuff!

Just an idea: Logfile name could be set as `x-server-instance-token` so that multiple logfiles could be opened at the same time

Commit:
Add logfile name to server instance token for debug functions
